whipper/common/config.py

```diff
@@ -89,10 +89,10 @@ def setReadOffset(self, vendor, model, release, offset):
 
         Strips the given strings of leading and trailing whitespace.
 
-        :param vendor:
-        :param model:
-        :param release:
-        :param offset:
+        :param vendor:  Drive vendor
```

Comment:
I think it'd be worth simply deleting the `:param vendor:` etc lines wholesale. The meaning of the parameters is already quite clear from context.